tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -38,12 +38,27 @@ struct spi_cs_control spi_cs = {
 
 #define STACK_SIZE 512
 #define BUF_SIZE 17
+#define BUF2_SIZE 36
+
+#if CONFIG_NOCACHE_MEMORY
+static const char TX_DATA[] = "0123456789abcdef\0";
+static __aligned(32) char buffer_tx[BUF_SIZE] __used
+	__attribute__((__section__(".nocache")));
+static __aligned(32) char buffer_rx[BUF_SIZE] __used
+	__attribute__((__section__(".nocache")));
+static const char TX2_DATA[] = "Thequickbrownfoxjumpsoverthelazydog\0";
```

Comment:
Please convert `TX_DATA` and `TX2_DATA` to `tx_data` and `tx2_data` respectively

Reply:
done